colossalai/inference/modeling/policy/nopadding_baichuan.py

```diff
@@ -21,14 +24,13 @@ def module_policy(self):
         policy = super().module_policy()
 
         decoder_attribute_replacement = {
-            "lm_head.weight": Parameter(
-                nn.functional.normalize(self.model.lm_head.weight).transpose(0, 1), requires_grad=False
-            ),
+            "lm_head.weight": Parameter(nn.functional.normalize(self.model.lm_head.weight), requires_grad=False),
```

Comment:
Wonder if the change aims at supporting TP?

Reply:
YES